Sources/PodToBUILD/BuildFile.swift

```diff
@@ -140,10 +140,12 @@ struct InsertAcknowledgementsTransform: SkylarkConvertibleTransform {
                     return [target]
                 }
 
+
                 let deps = target.acknowledgedDeps ?? [String]()
+                let externalDeps = deps.filter { $0.hasPrefix("//") }
                 let acknowledgement = AcknowledgmentNode(name: target.name,
                                                          license: podSpec.license,
-                                                         deps: deps)
```

Comment:
If depend on the subspec, `//IGListKit:Core`, will this correctly propagate all licensing? I thought for some reason, this was required.

Reply:
Yes since the other pod will begin with `//Vendor/IGListKit`